app/features/syllabus_generator/sample request.json

```diff
@@ -0,0 +1,49 @@
+{
```

Comment:
This sample request is not required for the main project. It must be in the README